docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -726,6 +726,12 @@ There are performance implications but they are typically mild due to hardware a
 
 ---
```

Comment:
The addition is on the wrong side of the section separator. The _Additional considerations_ should be placed before the footnote text.